content/docs/dataform-cli/index.md

```diff
@@ -86,6 +86,24 @@ To see the output of the compilation process as a JSON object, add the `--json`
 dataform compile --json
 ```
 
+### Variable injection
+
+Variables can be directly passed in to code by using the `--vars` flag. For example if you have the file `vars_example.sqlx` in your project that looks like:
+
+```sql
+config {
+  "type": "table"
+}
+
+select 1 as ${dataform.projectConfig.vars.exampleVar}
+```
```

Comment:
instead of giving another example of variable usage, link to the existing docs? https://docs.dataform.co/guides/configuration#configure-custom-compilation-variables

e.g. "If your project uses [custom compilation variables](link), you can pass their values using the `--vars` flag:"

Reply:
Ah nice